src/Samples/Sample.AspNetCore.SystemTests/Test/Helpers/PayexTrustlyInfo.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Sample.AspNetCore.SystemTests.Test.Helpers
+{
+    public class PayexTrustlyInfo : PayexInfo
+    {
+        public PayexTrustlyInfo()
```

Comment:
Codacy found an issue: [Remove this redundant constructor.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6074055)